templates/unbound.conf.erb

```diff
@@ -122,7 +122,7 @@ server:
     <%- end -%>
   <%- end -%>
 <%- end -%>
-<%= print_config('chroot', @chroot) -%>
+<%= print_config('chroot', @chroot, false, true) -%>
```

Comment:
I wonder if this wouldn't be clearer to have a separate if statement, just like on line 40.
```suggestion
<%= print_config('chroot', @chroot) -%>
```